src/Widgets/AddressWidget/AddressWidgetClass.ts

```diff
@@ -3,6 +3,7 @@ import { provideWidgetClass } from 'scrivito'
 export const AddressWidget = provideWidgetClass('AddressWidget', {
   attributes: {
     showLogo: 'boolean',
+    brandName: 'string',
```

Comment:
* [ ] Now editors have to set the same alt text three times. I don't think the added text attribute is worth it.